pants.toml

```diff
@@ -49,6 +49,7 @@ plugins = [
 pantsd_invalidation_globs.add = [
   "!*_test.py",
   "!BUILD",
+  "!src/python/pants_release/**",
```

Comment:
Ah, thanks, having to wait for pantsd to restart for any code change there has been frustrating. I didn't realise it was resolvable!